hw/ip/rv_dm/data/rv_dm.hjson

```diff
@@ -103,15 +103,144 @@
     regs:[
     ]
     mem: [
-      // Debug memory size (given as `items` below) must be a power of two.
+      // Expansion of debug memory within rv_dm used for DV purposes. The generated reg_top is not
+      // actually instantiated in RTL since the debug memory region is implemented inside the
+      // vendored-in module from the PULP project.
```

Comment:
I suppose this also needs to be updated?

Reply:
This part is actually up to date - we do not instantiated the reg_top in RTL since the registers are all implemented inside the PULP module.

Reply:
It will be nice to not have to check in unused code! :)

Reply:
yeah we have to find a way to not generate that `reg_top` - I believe CI fails right now if we don't check it in. there are other such dummy `reg_top`s in the tree, e.g. for `sram_ctrl`...